crates/nomination/src/lib.rs

```diff
@@ -236,9 +233,12 @@ impl<T: Config> Pallet<T> {
         ensure!(Self::is_opted_in(&vault_id)?, Error::<T>::VaultNotOptedInToNomination);
 
         let currency_id = ext::vault_registry::get_collateral_currency::<T>(&vault_id)?;
-
         let amount = Amount::new(amount, currency_id);
 
+        // we're unable to maintain the max nomination ratio if the vault has no issued tokens
+        let vault = ext::vault_registry::get_active_vault_from_id::<T>(&vault_id)?;
+        ensure!(!vault.issued_tokens.is_zero(), Error::<T>::VaultNoIssuedTokens);
```

Comment:
I don't see why we can't maintain the max nomination ratio if there are no issued tokens, since the ratio is `nominator_collateral : vault_collateral`, which shouldn't have anything to do with issued tokens.

Reply:
Yep you're right, I can't recall exactly what the issue was here - I'll close for now.